Tella/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -240,9 +240,18 @@ If you want to take a picture or a video, proceed in the following way:\
 "Setting_SettServer_Selection_Message" = "If you are not sure, please ask your organization.";
 "Setting_SettServer_TellaWeb" = "TELLA WEB";
 "Setting_SettServer_Uwazi" = "UWAZI";
+"Setting_SettServer_GDrive" = "GOOGLE DRIVE"; 
 "Setting_SettServer_No_Internet" = "No Internet connection. Try again when you are connected to the Internet.";
 "Setting_SettServer_Server_URL_Incorrect" = "Error: The server URL is incorrect";
-
+"Setting_SettServer_UnavailableConnections_Title" = "Unavailable connections";
```

Comment:
What do you think about updating the keys to this :


"Setting_SettServer_UnavailableConnections_Subhead" = "Unavailable connections";
"Setting_SettServer_UnavailableConnections_Expl" = "For the following categories, only one connection can be enabled at a time. To add a new connection, please delete the current one by going to Connections Settings.";


"Settings_SettServer_DeleteTellaConnection_SheetTitle" = "Delete %@ server?";
"Settings_SettServre_DeleteConnection_SheetTitle" = "Delete \"%@\" connection?";
"Settings_SettServer_DeleteConnection_SheetExpl" = "if you delete this server, all draft and submitted forms will be deleted from your device.";
"Settings_SettServer_DeleteUwaziConnection_SheetExpl" = "If you delete this server, all draft and submitted entities will be deleted from your device. Delete anyway?";